app/templates/public/index.html

```diff
@@ -8,13 +8,21 @@
 {% block title %}College Predictor 2020{% endblock %}
 
 {% block main %}
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="
+  Welcome to College Predictor. College Predictor is a website which helps for IIT, JEE Aspirants to predict or find college and Branch you are likely to get into based on you percentile, quota, gender for JEE Mains 2020">
+  <meta name="keywords" content="IIT, JEE, College, Predictor, Engineering, 2020">
+  <meta name="author" content="Niharika Kabra">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Try to add some more meta stuff if possible !

Reply:
ok i have researched and added as much as i can still i would research and add

Reply:
Hello I have bit researched and made change in meta description and added title to the link. I think this much is the meta information i can add for on page seo.

Reply:
Hello I researched about on page seo and did much I can in project HTML page can you accept my pull request